src/utils/navigation.ts

```diff
@@ -96,7 +96,7 @@ export const NAVIGATION: Navigation = {
     nextPage: []
   },
   [config.UPDATE_BENEFICIAL_OWNER_STATEMENTS_URL]: {
```

Comment:
any change to registration page for this? Probably it's already right but double checking that no changes needed for registration would be good.

Reply:
Just checked and no changes have been made to registration. Both urls lead to the same view as expected